### ISSUE TITLE
pygobject not found

### ISSUE THREAD

**Issue by cinghiopinghio** [coderkun/mcg#31]
I would love to try `mcg` but I encounter the following error at runtime:
```
Traceback (most recent call last):
  File "/usr/bin/mcg", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3142, in <module>
    @_call_aside
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3126, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 3155, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 664, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 981, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 867, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'pygobject' distribution was not found and is required by mcg
```
I installed from the AUR in archlinux.

Any hints?

**Comment by coderkun** [comment]
Sorry for taking so long, @cinghiopinghio. I have removed “pygobject” as dependency since it is not a python dependency and pushed this fix as bugfix version 1.2.2. If you get the latest code, you should be able to start the application. Make sure you have “python-gobject” installed.

If you don’t want to build the application yourself, feel free to use the binary packages from [coderkun.de/arch](https://coderkun.de/arch).

**Comment by cinghiopinghio** [comment]
I confirm that the build works now (except that the AUR package for arch linux have the wrong sha256).
I have a different issue now, I will open a new thread.
